### PR TITLE
dbus: Update dd_get_owner

### PR DESCRIPTION
**Pull request by mkutlak**
Updates `dd_get_owner` according to changes in https://github.com/abrt/libreport/pull/570

Signed-off-by: Martin Kutlak <mkutlak@redhat.com>

### CODE REVIEW

File: abrt.spec.in [hunk 1]
```diff
@@ -43,7 +43,7 @@
     %define docdirversion -%{version}
 %endif
 
-%define libreport_ver 2.10.0
```
Comment (mkutlak) [inline abrt.spec.in]: The PR will fail until the PR in libreport is merged and new libreport is released.

**Comment by mgrabovsky** [comment]
What's the status of this (and the related libreport PR)? Ready for re-review?

**Comment by mkutlak** [comment]
Depends on: https://github.com/abrt/libreport/pull/570

**Comment by mkutlak** [comment]
> What's the status of this (and the related libreport PR)? Ready for re-review?

I guess. 

I don't see myself working on this in the future.

**Comment by xsuchy** [comment]
@mgrabovsky will take over, but we will wait till @michalfabik finish his PR with locking

**Comment by mgrabovsky** [comment]
Took over in #1484.

**Pull request closed without merging**